portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +23,16 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+//    Arraylist<String> comments = new Arraylist<String>();
+//     comments.add("I am a pretty girl");
+//     comments.add("I am a cute girl");
+//     comments.add("I am a phenomenal girl");
```

Comment:
You can probably delete these too, since it doesn't look like you're using them anyway.